rmarkdowns/Week_4.Rmd

```diff
@@ -261,7 +261,107 @@ For your challenges this week, create the necessary code chunks below. You can c
 
 **Challenge 3: Create an interactive map using the `leaflet` function of your target area and target genus/species.** 
 
```

Comment:
One note for future, it is helpful to break up your code into smaller code chunks. This makes it easier to run (my computer wasn't able to run the whole chunk below, I had to break it up to run it. Also, it makes it easier to test your code along the way to make sure things are working as you go. It also just helps to organize things :)